### ISSUE TITLE
Recipes print run-name CLI commands that fail (run ids are generated)

### ISSUE THREAD

**Issue by abhay-codes07** [rajudandigam/agent-inspect#144]
## Summary

Two recipes print copy-paste CLI commands that always fail, because they pass the `inspectRun` run name where the CLI expects a run id. Run ids are always generated (`run_xxx`), and the CLI resolves a run reference strictly as `<traceDir>/<runId>.jsonl` (`packages/cli/src/read-run.ts`), with no name lookup.

Affected recipes:

- `examples/recipes/shareable-bundle-basic`: prints `Run id: bundle-recipe-run` (which is the run name, not the id) and `npx agent-inspect bundle bundle-recipe-run ...`
- `examples/recipes/what-report-inspect`: prints `what what-report-recipe`, `report what-report-recipe ...` commands

`expected-output.txt` for shareable-bundle-basic pins the wrong output, so `pnpm recipes:check` does not catch this (it validates layout, not command correctness).

## Repro

```bash
cd examples/recipes/what-report-inspect
pnpm start
# recipe prints:
#   npx agent-inspect what what-report-recipe --dir .../.agent-inspect-runs
npx agent-inspect what what-report-recipe --dir ./.agent-inspect-runs
```

Output:

```text
Run not found: what-report-recipe
Trace directory: ./.agent-inspect-runs
```

Exit code 1. Same trace inspected with the generated id (`what run_8IIiuO5KS7`) exits 0.

```bash
cd examples/recipes/shareable-bundle-basic
pnpm start
npx agent-inspect bundle bundle-recipe-run --dir ./.agent-inspect --out ./bundle-out --json
```

Output:

```json
{
  "error": "Run \"bundle-recipe-run\" was not found in the trace directory.",
  "ok": false
}
```

## Why it happens

`inspectRun(name, fn, options)` takes a run name as its first argument; the persisted file is named after the generated run id. Both recipes assign the name to a variable called `runId` and interpolate it into the printed commands, so the commands reference a file that never exists.

Other recipes are not affected: `langgraph-callback-local` passes an explicit `runId` to `AgentInspectCallback` (which supports it), and the import/fixture recipes control their ids directly.

## Proposed fix

Keep this examples-only and freeze-safe: after `inspectRun` completes, discover the generated run id from the trace directory (newest `*.jsonl` basename, the same pattern `adpa-codebase-change-trace` in #142 uses) and print that in the `Run id:` line and the commands. Update the READMEs and `expected-output.txt` to match (with a placeholder note for the generated id).

A CLI-side run-name lookup would be a bigger behavior change (names are not unique across runs) and does not seem freeze-appropriate; happy to split that discussion out if there is interest.

I can take this; fix PR to follow shortly.
